### DISCUSSION TITLE
How to use second querty layout on colemak-DH?

### DISCUSSION THREAD

**Discussion by alex-morgun** [manna-harbour/miryoku#214] [Q&A] [answered]
I have Corne with zmk on Colemak-DH

How to use the Ukrainian layout? It is mapped incorrectly and works only if the keyboard is set to qwerty

I use Linux, is there a way to map it correctly?

**Comment by manna-harbour** [comment, marked as answer]
You can change the Base layer to QWERTY, see https://github.com/manna-harbour/miryoku_zmk#alphas. Or you can switch between Base and Extra, which is QWERTY by default, see https://github.com/manna-harbour/miryoku/tree/master/docs/reference#additional-features.

Also note:
https://github.com/manna-harbour/miryoku/issues/35
https://github.com/manna-harbour/miryoku/discussions/85
https://github.com/manna-harbour/miryoku/discussions/206
https://github.com/manna-harbour/miryoku/discussions/53
https://github.com/manna-harbour/miryoku/discussions/197
